.circleci/create_pypirc.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/env/python
+import sys
+
+def main(argv):
```

Comment:
at some point we might want to make this as an argparser. But looks good for now

Reply:
Yeah and if you know offhand of and easier templating mechanism let me know. I used to have a trick using just bash but I forgot it.